browser-test/src/trusted_intermediary.test.ts

```diff
@@ -753,4 +753,233 @@ describe('Trusted intermediaries', () => {
+
+      // There should be no ellipses
+      expect(await page.innerHTML('.usa-pagination__list')).not.toContain(
+        'usa-pagination__overflow',
+      )
+    })
+
+    it('shows 2 pages when there are 12 clients', async () => {
+      const {page, tiDashboard} = ctx
+      await loginAsTrustedIntermediary(page)
+      await tiDashboard.gotoTIDashboardPage(page)
+      await waitForPageJsLoad(page)
+
+      await tiDashboard.createMultipleClients('myname', 11)
+
+      // Page 1 should still only show 10 clients
+      const cardCount = await page.locator('.usa-card__container').count()
+      expect(cardCount).toBe(10)
+
+      expect(await page.innerHTML('.usa-pagination__list')).not.toContain(
+        'usa-pagination__previous-page',
+      )
+
+      expect(await page.innerHTML('.usa-pagination__list')).toContain(
+        'usa-pagination__next-page',
+      )
+
+      await tiDashboard.expectPageNumberButtonOrNot('1', true)
+      await tiDashboard.expectPageNumberButtonOrNot('2', true)
+
+      expect(await page.innerHTML('.usa-current')).toContain('1')
+
+      expect(await page.innerHTML('.usa-pagination__list')).not.toContain(
+        'usa-pagination__overflow',
+      )
+
+      // Going to page 2
+      await page.click('[aria-label=Page2]')
+
+      const page2CardCount = await page.locator('.usa-card__container').count()
+      expect(page2CardCount).toBe(1)
+
+      // Now there should be a 'Previous' button
+      expect(await page.innerHTML('.usa-pagination__list')).toContain(
+        'usa-pagination__previous-page',
+      )
+
+      expect(await page.innerHTML('.usa-pagination__list')).not.toContain(
+        'usa-pagination__next-page',
+      )
+
+      await tiDashboard.expectPageNumberButtonOrNot('1', true)
+      await tiDashboard.expectPageNumberButtonOrNot('2', true)
+
+      expect(await page.innerHTML('.usa-current')).toContain('2')
+    })
+
+    it('shows 7 pages and no ellipses when there are 65 clients ', async () => {
+      const {page, tiDashboard} = ctx
+      await loginAsTrustedIntermediary(page)
+      await tiDashboard.gotoTIDashboardPage(page)
+      await waitForPageJsLoad(page)
+
+      await tiDashboard.createMultipleClients('myname', 65)
+
+      await tiDashboard.expectPageNumberButtonOrNot('1', true)
+      await tiDashboard.expectPageNumberButtonOrNot('2', true)
+      await tiDashboard.expectPageNumberButtonOrNot('3', true)
+      await tiDashboard.expectPageNumberButtonOrNot('4', true)
+      await tiDashboard.expectPageNumberButtonOrNot('5', true)
+      await tiDashboard.expectPageNumberButtonOrNot('6', true)
+      await tiDashboard.expectPageNumberButtonOrNot('7', true)
+      await tiDashboard.expectPageNumberButtonOrNot('8', false)
+
+      // Going to page 7
+      await page.click('[aria-label=Page7]')
+      expect(await page.innerHTML('.usa-current')).toContain('7')
+
+      expect(await page.innerHTML('.usa-pagination__list')).not.toContain(
```

Comment:
suggestion (non-blocking): add a comment about what `usa-pagination_overflow` is. I think elipses?

Reply:
Added a comment about ellipses whenever `usa-pagination_overflow` is used.